src/main/java/JSON/Json.file

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Here I see 3 classes. Requirements were to create at least for 5 classes.

Reply:
Hi Zhyldyz, there are five entities with hierarchical relationships. Booking is the central entity.

Reply:
Ok got it.